pr_agent/tools/pr_description.py

```diff
@@ -74,7 +74,7 @@ async def run(self):
 
             await retry_with_fallback_models(self._prepare_prediction)
 
-            get_logger().info(f"Preparing answer {self.pr_id}")
+            get_logger().info(f"Preparing description regarding {self.pr_id}")
             if self.prediction:
                 self._prepare_data()
             else:
```

Comment:
**Suggestion:** Add a log message to handle the case where `self.prediction` is `None` after retrying, which would be helpful for debugging.
```suggestion
            if self.prediction:
                self._prepare_data()
            else:
                get_logger().error("Prediction failed to prepare", extra={"pr_id": self.pr_id})
```